docs/nidcpower/class.rst

```diff
@@ -739,7 +739,7 @@ fetch_multiple
                 .. note:: When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.
 
 
-            :type timeout: float in seconds or datetime.timedelta
+            :type timeout: float in seconds or hightime.timedelta
```

Comment:
Since we document this can take a float, why wouldn't we document this can take a `datetime.timedelta`?
Applies to all input parameters.

Reply:
Yup, we should. I will update. I'll also reorder it as:
```
hightime.timedelta or datetime.timedelta or float in seconds
```

Reply:
Ok, remember there are several that take integer in milliseconds, since you'll be copying and pasting.

I've said this before - I'd rather not even mention the fact that you can pass a number and I sort of regret it.

Reply:
> remember there are several that take integer in milliseconds

Good point, thanks for reminding me. I made the appropriate changes.

> I've said this before - I'd rather not even mention the fact that you can pass a number and I sort of regret it.

Agreed. There is an issue for it - https://github.com/ni/nimi-python/issues/1388